storage_drivers/storage_drivers.go

```diff
@@ -64,15 +64,17 @@ type ESeriesStorageDriverConfig struct {
 	CommonStorageDriverConfig
 
 	//Web Proxy Services Info
-	WebProxy_Hostname string `json:"webProxyHostname"`
-	Username          string `json:"username"` //rw
-	Password          string `json:"password"` //rw
+	WebProxyHostname string `json:"webProxyHostname"`
+	WebProxyPort     string `json:"webProxyPort"`     // optional
+	WebProxyInsecure bool   `json:"webProxyInsecure"` // optional
+	Username         string `json:"username"`         //rw
+	Password         string `json:"password"`         //rw
 
 	//Array Info
-	Controller_A     string `json:"controllerA"`
-	Controller_B     string `json:"controllerB"`
-	Password_Array   string `json:"passwordArray"`   //optional
-	Array_Registered bool   `json:"arrayRegistered"` //optional
+	ControllerA     string `json:"controllerA"`
+	ControllerB     string `json:"controllerB"`
+	PasswordArray   string `json:"passwordArray"`   //optional
+	ArrayRegistered bool   `json:"arrayRegistered"` //optional
 
 	//Host Networking
 	HostData_IP string `json:"hostData_IP"` //for iSCSI can be either port if multipathing is setup
```

Comment:
I left `HostData_IP` intact to avoid breaking existing configuration files. But I guess I can change the Go field name, and leave the JSON tag as is. Thoughts?